egs/ami/s5/local/run_cleanup_segmentation.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
```

Comment:
Should there be results at the top of this file?   If there are any comparisons of different settings you could consider including them here, or at least point from here to the correct place.